e2e/scenario_helpers_test.go

```diff
@@ -121,42 +121,46 @@ func maybeSkipScenario(ctx context.Context, t *testing.T, s *Scenario) {
 			t.Fatalf("could not find image for %q: %s", t.Name(), err)
 		}
 	}
-	t.Logf("running scenario %q with vhd: %q, tags %+v", t.Name(), vhd, s.Tags)
+	t.Logf("running scenario vhd: %q, tags %+v", vhd, s.Tags)
```

Comment:
the scenario name wasn't useful ?

Reply:
each log message is associated with a test. It's just a duplication of the available information.